examples/express/README.md

```diff
@@ -44,59 +46,61 @@ To start collecting and exporting tracing data from the app it requires
+
+All that's left is to run it:
 
 ```shell
-npm run server
-# in a separate terminal:
-npm run client
+npm run server:collector
+# In a separate terminal:
+npm run client:collector
+
 ```
 
 See the exact commands in [package.json](./package.json).
-If unsure about the values to use for `.env.example` but familiar with configuring OpenTracing SDK, one can alternatively also use [.env.opentracing.example](./.env.opentracing.example) as the basis - the OTel configuration will be automatically derived from that in this example(see [utils.js](./utils.js) for the conversions).
 
 ### Instrumented via OTel SDK, Jaeger Exporter
 
-There's an [tracer setup](./tracer.jaeger.js) to showcase replacing the default OTLP/HTTP exporter with one that exports in Jaeger format. The configuration is similar to the plain OTel SDK setup, but
-
-1. `@opentelemetry/exporter-jaeger` package has to be installed and the factory initiating the Exporter has to be passed to `startTracing`;
-2. The endpoint(`OTEL_EXPORTER_JAEGER_ENDPOINT`) for Splunk's APM has to be replaced by one that accepts Jaeger formatted tracing data: `https://ingest.<realm>.signalfx.com/v2/trace`.
+There's an [tracer setup](./tracer.jaeger.js) to showcase replacing the default OTLP/gRPC Exporter with one that exports in Jaeger format. The configuration is similar to the plain OTel SDK setup, but the Exporter(`OTEL_TRACES_EXPORTER`) has to be replaced by the default Jaeger Exporter which is bundled with the SDK: `jaeger-thrift-http`.
 
 That's it! To run the example with the Jaeger Exporter:
 
 ```shell
 npm run server:jaeger
-# in a separate terminal:
+# In a separate terminal:
 npm run client:jaeger
 ```
 
-### Instrumented via OTel SDK, exporting to locally running collector
+### Instrumented via OTel SDK, exporting to Splunk APM using Jaeger Exporter
 
-Instead of sending telemetry data directly to Splunk APM API, one can also use OTel Collector to as a forwarder or an endpoint.
-There is an exmample [.env](./.env.collector) file included, which following commands use:
+Instead of using OTel Collector to forward the telemetry data to Splunk APM, one can also send it directly from the application.
+Once `.env` file is created(see file [.env.jaeger-splunk](./.env.jaeger-splunk) for an example) and the access token is substituted into the file. Run it using that:
 
 ```shell
-# make sure the collector is running
-docker run --name otel-collector -d -p 55681:55681 otel/opentelemetry-collector
-# run the example server
-npm run server:collector
-# in a separate terminal:
-npm run client:collector
+npm run server
+# In a separate terminal:
+npm run client
 ```
 
+If unsure about the values to use for `.env` but familiar with configuring OpenTracing SDK, one can alternatively also use [.env.opentracing](./.env.opentracing) as the basis - the OTel configuration will be automatically derived from that in this example(see [utils.js](./utils.js) for the conversions).
```

Comment:
```suggestion
If are not sure about the values to use for `.env`, but you're familiar with configuring the OpenTracing SDK, you can also use [.env.opentracing](./.env.opentracing) as the baseline configuration: The OTel configuration is automatically derived from the OpenTracing settings in the example (see [utils.js](./utils.js) for the conversions).
```

Reply:
@Rauno56 This is pretty cool. Could users use this method when migrating?

Reply:
Not... directly. The example also logs the configuration out so they will get an idea of the resulting configuration, but there's no "automatic" way implemented.

They could use the function from `utils.js` themselves though, but I think just reading the code will be informative enough for them to do the conversion once and for all.